cookbooks/bcpc-hadoop/attributes/phoenixqs.rb

```diff
@@ -0,0 +1,10 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2
+
+default['bcpc']['hadoop']['phoenix']['phoenixqs']['username'] = 'phoenixqs'
```

Comment:
Can we add an attribute so that the query server can be enabled/disabled through ``chef`` without performing these actions manually.

Reply:
Enablement/disablement is done on a role basis.  For maintenance purposes we would just shut down chef-client